lib/anubis.go

```diff
@@ -476,6 +484,12 @@ func (s *Server) PassChallenge(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Adjust cookie path if base prefix is not empty
+	cookiePath := "/"
+	if anubis.BasePrefix != "" {
+		cookiePath = anubis.BasePrefix + "/"
+	}
```

Comment:
This bit probably needs real-world testing to be on the safe side. Let me know if you want root access to one of my test servers with apache and gitweb set up!